### PR TITLE
Module changes

### PR DESCRIPTION
**Pull request by matzew**
changes ! :-) 

renamed modules as discussed, and fixing (somewhat) related pom issues

### CODE REVIEW

File: operator/pom.xml [hunk 1]
```diff
@@ -86,7 +86,7 @@
                     <images>
                         <image>
                             <alias>watcher</alias>
-                            <name>docker.io/streamzi/cef-watcher</name>
```
Comment (hhiden) [inline operator/pom.xml]: cef stands for cloud-event-flow, should it be changed to ef-watcher?

Comment (matzew) [reply operator/pom.xml]: @hhiden I'd think we go with `evenflow`, as full name. but `ef` is OK for me as well

Comment (sjwoodman) [reply operator/pom.xml]: So eventflow-operator ?

Comment (hhiden) [reply operator/pom.xml]: Agreed eventflow is better

**Comment by matzew** [comment]
@sjwoodman @hhiden 

https://github.com/project-streamzi/cloudevent-flow/pull/68/files#diff-e6a94af5326732d23cb36787a1186b0dR11

IT's actually `eventflow-operator` - the file you are commenting on, is gone. I think it's due to a bunch of several commit, that it was present, before I did a little rebase

**Comment by sjwoodman** [comment]
:) Good work @matzew

**Comment by matzew** [comment]
> Good work

I take that as approved ;-)

**Pull request merged**